sdk/azcore/internal/pollers/util.go

```diff
@@ -78,6 +78,11 @@ func Failed(s string) bool {
 	return strings.EqualFold(s, StatusFailed) || strings.EqualFold(s, StatusCanceled)
 }
 
+// Succeeded returns true if the LRO's state is terminal success.
+func Succeeded(s string) bool {
+	return strings.EqualFold(s, StatusSucceeded)
```

Comment:
Do we not control the constant value here? Why would it change case?

Reply:
Talked with @jhendrixMSFT - the value of the string is part of the contract but the casing of the string isn't fixed from other services.